public/components/event_analytics/explorer/events_views/data_grid.tsx

```diff
@@ -41,21 +41,26 @@ interface DataGridProps {
   startTime: string;
   endTime: string;
   storedSelectedColumns: IField[];
+  formatGridColumn?: (columns: EuiDataGridColumn[]) => EuiDataGridColumn[];
+  OUIDataGridProps?: Partial<EuiDataGridProps>;
```

Comment:
nit. oui uses `Oui` in camelcases
```suggestion
  OuiDataGridProps?: Partial<EuiDataGridProps>;
```